### ISSUE TITLE
FlatList wont Scroll Till end of screen

### ISSUE THREAD

**Issue by Victorvikson1996** [osdnk/react-native-reanimated-bottom-sheet#385]

**Comment by krushalikevadiya** [comment]
same issue.

**Comment by Victorvikson1996** [comment]
I think you need to check your styling make show your scrollView is under SafeAreaView

**Comment by akinncar** [comment]
for me, add `contentContainerStyle` to the FlatList with the sum of all items height + use FlatList from react-native-gesture-handles solved the problem

**Comment by Victorvikson1996** [comment]
> 

Yes that's the best solution to that Thank you so much @akinncar
